site-scrapers/TuftsMC/index.js

```diff
@@ -0,0 +1,135 @@
+const https = require("https");
+const file = require("../../lib/file");
+const fetch = require("node-fetch");
+const { site, requestInit } = require("./config");
+
+module.exports = async function GetAvailableAppointments(
+    browser,
+    pageService = defaultPageService()
+) {
+    console.log(`${site.name} starting.`);
+    const webData = await ScrapeWebsiteData(browser, pageService);
+    console.log(`${site.name} done.`);
+    return {
+        ...site,
+        ...webData,
+        timestamp: new Date(),
+    };
+};
+
+function defaultPageService() {
+    return {
+        async fetchAvailability() {
+            return fetchAvailability();
+        },
+    };
+}
+
+async function ScrapeWebsiteData(browser, fetchService) {
+    // Initialize results to no availability
+    const results = {
+        availability: {},
+        hasAvailability: false,
+    };
+
+    /* Note:
+        site has next button if 'button[data-test="desktop-controls-right-arrow"]'
+        is not disabled
+    */
+
+    const response = await fetchService.fetchAvailability();
+
+    results.availability = {
+        ...results.availability,
+        ...parseResponse(response),
+    };
+    // }
+
+    results.hasAvailability = !!Object.keys(results.availability).length;
+
+    if (process.env.DEVELOPMENT) {
+        file.write(
```

Comment:
our results are already written to out.json (or the new out_no_browser.json or whatever it's called), so I think this is unnecessary

Reply:
Hmmm. I put that in there because scrapers.js wasn't writing out.json. Maybe it has been put back in? I'll remove this.

Reply:
That code originates earlier than the browser/browserless split. I think I mistakenly
put it in the scraper instead of the test file because I didn't see anything in out.json. I think I was momentarily confused about which context I was running in -- unit tests don't get passed through the scraper.js post-processing when unit testing. Sometimes I think I get too deep into the trees, like into the xylem and phloem, that I really cannot see the forest.

I have removed it.